### PR TITLE
TechDocs: Added more plugins to mkdocs

### LINKED ISSUE #2068: [TechDocs] Add PlantUML support to the techdocs container
Suggested extension: https://github.com/mikitex70/plantuml-markdown

But feel free to suggest other extensions that are compatible with MkDocs

### PR DESCRIPTION
**Pull request by hooloovooo**
closes: #2068

### CODE REVIEW

File: packages/techdocs-container/techdocs-core/requirements.txt [hunk 1]
```diff
@@ -2,6 +2,13 @@
 # Note: if you update this, also update `install_requires` in setup.py
 # https://github.com/mkdocs/mkdocs
 mkdocs==1.1.2
+mkdocs-material==5.3.2
+mkdocs-monorepo-plugin==0.4.5
+markdown-katex==202008.1025
```
Comment (emmaindal) [inline packages/techdocs-container/techdocs-core/requirements.txt:7]: wait with this one

File: packages/techdocs-container/techdocs-core/setup.py [hunk 2]
```diff
@@ -28,7 +28,14 @@
     license='Apache-2.0',
     python_requires='>=3.7',
     install_requires=[
-        'mkdocs>=1.1.2'
+        'mkdocs>=1.1.2',
+        'mkdocs-material==5.3.2',
+        'mkdocs-monorepo-plugin==0.4.5',
+        'markdown-katex==202008.1025',
```
Comment (emmaindal) [inline packages/techdocs-container/techdocs-core/setup.py:34]: wait with this one

File: packages/techdocs-container/techdocs-core/setup.py [hunk 3]
```diff
@@ -18,7 +18,7 @@
 
 setup(
     name='mkdocs-techdocs-core',
-    version='0.0.3',
+    version='0.0.4',
```
Comment (emmaindal) [inline packages/techdocs-container/techdocs-core/setup.py:21]: can we add a changelog to the README to keep track of changes in the versions

File: packages/techdocs-container/techdocs-core/src/core.py [hunk 4]
```diff
@@ -90,4 +90,13 @@ def on_config(self, config):
         }
         config["markdown_extensions"].append("pymdownx.tilde")
 
+        config["markdown_extensions"].append("markdown_inline_graphviz")
+        config["markdown_extensions"].append("plantuml_markdown")
+
+        config["markdown_extensions"].append("markdown_katex")
+        config["mdx_configs"]["markdown_katex"] = {
```
Comment (emmaindal) [inline packages/techdocs-container/techdocs-core/src/core.py:97]: wait with this one

File: packages/techdocs-container/Dockerfile [hunk 5]
```diff
@@ -12,9 +12,18 @@
...
-FROM python:3.7.7-alpine3.12
+FROM python:3.8-alpine
+
+
+RUN apk update && apk --no-cache add gcc musl-dev openjdk11-jdk curl graphviz ttf-dejavu fontconfig
+
+RUN curl -L http://sourceforge.net/projects/plantuml/files/plantuml.1.2020.16.jar/download > /opt/plantuml.jar
+RUN pip install --upgrade pip && pip install mkdocs-techdocs-core==0.0.3
+
+# Copy plantuml jar file
```
Comment (hooloovooo) [inline packages/techdocs-container/Dockerfile:23]:
```suggestion
# Create script to call plantuml.jar from a location in path
```

**Review by emmaindal — APPROVED** [review]

**Pull request merged**